test/org/zalando/stups/magnificent/api_test.clj

```diff
@@ -0,0 +1,134 @@
+(ns org.zalando.stups.magnificent.api-test
+  (:require [clojure.test :refer :all]
+            [org.zalando.stups.magnificent.external.team :as team]
+            [org.zalando.stups.magnificent.external.user :as user]
+            [org.zalando.stups.magnificent.external.account :as account]
+            [org.zalando.stups.magnificent.api :as api]))
+
+(defn track
```

Comment:
we should definitely consider using midje for testing. This library is just awesome: much more readable + great mocking capabilities